telegram/utils/request.py

```diff
@@ -224,14 +229,38 @@ def _request_wrapper(self, *args, **kwargs):
         else:
             raise NetworkError('{0} ({1})'.format(message, resp.status))
 
-    def get(self, url, timeout=None):
+    def _parse_retries(self, retries):
+        """
+        Builds a :obj:`telegram.vendor.ptb_urllib3.urllib3.util.retry.Retry object.
+        If it's a positive integer, it's used as the total amount of retries, and a new Retry
+        object with a sane default for backoff (0.1s) is created. Else, retrying is disabled.
+
+        Args:
+            retries (:obj:`int`): Retries parameter.
+
+        Returns:
+          A :obj:`telegram.vendor.ptb_urllib3.urllib3.util.retry.Retry` object, or False if
+          retrying is disabled.
+
+        """
+        # TODO: allow configuring the backoff. Ideally we would just let the programmer use the
+        # Retry class for fine-tuning, but that's inside the "private" urllib3 package.
+
+        if isinstance(retries, int) and retries > 0:
+            return Retry(total=retries, backoff_factor=0.1)
```

Comment:
That looks like a reasonable default. :+1:

Reply:
Don't we need to specify a `status_forcelist`? [docs](https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#module-urllib3.util.retry)

Reply:
1. What http error codes should be retried in your opinion?
2. I'm not sure I want to retry on http errors. IMHO, It should be a much higher level logic decision, outside the control of urllib3.